pandas/tslib.pyx

```diff
@@ -2722,6 +2722,12 @@ class Timedelta(_Timedelta):
     __pos__ = _op_unary_method(lambda x: x, '__pos__')
     __abs__ = _op_unary_method(lambda x: abs(x), '__abs__')
 
+
+# Resolution is in nanoseconds
+Timedelta.min = Timedelta(np.iinfo(np.int64).min+1, 'ns')
+Timedelta.max = Timedelta(np.iinfo(np.int64).max, 'ns')
+Timedelta.resolution = Timedelta(1, 'ns')
```

Comment:
``.resolution`` is only used in the partial string matching, which expandas ``6 days`` to be essentially ``7 days`` - 1 ns. This is the same as what partial string matching does on strings for datetimes